clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -50,20 +50,23 @@ interface NexusIqService {
             password: String? = null,
             client: OkHttpClient? = null
         ): NexusIqService {
-            val authenticatedClient = (client ?: OkHttpClient()).newBuilder().apply {
-                if (user != null && password != null) {
-                    addInterceptor { chain ->
-                        val request = chain.request()
-                        val authenticatedRequest = request.newBuilder()
-                            .header("Authorization", Credentials.basic(user, password))
-                            .build()
-                        chain.proceed(authenticatedRequest)
+            val nexusIqClient = (client ?: OkHttpClient()).newBuilder().apply {
+                addInterceptor { chain ->
+                    val request = chain.request()
+                    val requestBuilder = request.newBuilder()
+                        .header("X-CSRF-TOKEN", "ORT")
```

Comment:
Sorry for not noticing earlier, but instead of `"ORT"` you should use the `ORT_NAME` constant. Similar below.

Reply:
Not possible as `org.ossreviewtoolkit.utils.ORT_NAME` is not available in this module.
Not sure if we should even use `ORT` then, or something more generic.

Reply:
Good point. Reading through https://en.wikipedia.org/wiki/Cross-site_request_forgery, "The CSRF token itself should be unique and unpredictable. It may be generated randomly, or it may be derived from the session token using HMAC".

Reply:
Replaced it with a random UUID.